client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -224,6 +251,22 @@ const getAddressErrors = ( addressData, countriesData ) => {
 		}
 	}
 
+	if ( shouldValidatePhone ) {
+		// EasyPost requires an origin phone number for international shipments.
+		// This validation ensures that EasyPost will accept it, even though the phone may be invalid.
+		if ( ! phone ) {
+			errors.phone = translate( 'Origin phone number is required for international shipments' );
+		} else if (
+			10 !==
+			phone
+				.split( /\D+/g )
+				.join( '' )
+				.replace( /^1/, '' ).length
+		) {
+			errors.phone = translate( 'Invalid phone number' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Invalid Phone Number' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-intl-labels-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).